learning/split/spynnaker.cfg

```diff
@@ -0,0 +1,2 @@
+[Mode]
+violate_1ms_wall_clock_restriction = True
```

Comment:
Continuing the theme from the other review, I don't think this particular cfg file is needed; the others are because of the combination of timestep=0.1 and time_scale_factor=1.

Reply:
Agreed - I would say it can be left anyway as it doesn't enforce this, only enable it.